code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -307,31 +307,29 @@
 				return 1
 			apply_damage(damage, BRUTE, affecting, armor_block)
 
-		if(M.a_intent == INTENT_DISARM) //Always drop item in hand, if no item, get stun instead.
+		if(M.a_intent == INTENT_DISARM) //Always drop item in hand on first go.  If no item exists, try to shove them back.  If you share the tile with the target, slam them directly into the ground to stun them and slightly damage them.
 			var/obj/item/I = get_active_held_item()
 			if(I && dropItemToGround(I))
 				playsound(loc, 'sound/weapons/slash.ogg', 25, 1, -1)
 				visible_message(span_danger("[M] disarmed [src]!"), \
 						span_userdanger("[M] disarmed [src]!"))
-			else
-				//yogs start
-				var/obj/item/bodypart/affecting = get_bodypart(ran_zone(M.zone_selected))
-				if(!affecting)
-					affecting = get_bodypart(BODY_ZONE_CHEST)
-				var/armour = run_armor_check(affecting, MELEE)
-				if(prob(armour))
-					to_chat(M, span_notice("[src]'s armour shields the blow!"))
-					return
-				playsound(loc, 'sound/weapons/pierce.ogg', 25, 1, -1)
-				if(armour > 0)
-					Paralyze(50 + armour)
-				else
-					Paralyze(50)
-				//yogs end
-				log_combat(M, src, "tackled")
-				visible_message(span_danger("[M] has tackled down [src]!"), \
-					span_userdanger("[M] has tackled down [src]!"))
-
+		else if(get_dist(src, M) != 0)
```

Comment:
mirror what TG does, this triggers if your intent isn't set to disarm

Reply:
Done